azure/sandbox-template.json

```diff
@@ -170,9 +202,6 @@
                     },
                     "keyVaultResourceGroup": {
                         "value": "[parameters('sharedManagementResourceGroup')]"
-                    },
-                    "serverFarmId": {
-                        "value": "[reference('sandbox-app-service-plan').outputs.appServicePlanId.value]"
                     }
                 }
             },
```

Comment:
Needs dependency on resource group (not sure why the existing dependency is there)